packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -146,10 +148,10 @@ function testFixtures(options?: RollupLwcOptions) {
                     )
                 );
             } catch (_err: any) {
-                if (_err.name === 'AssertionError') {
+                if (_err?.name === 'AssertionError') {
                     throw _err;
                 }
-                err = _err.message;
+                err = _err?.message || 'An empty error occurred?!';
```

Comment:
The error when `await import(compiledFixturePath)` fails has no error message. Not sure why.